### DISCUSSION TITLE
глубина JSON в POST

### DISCUSSION THREAD

**Discussion by tvideo** [totumonline/totum-ru-issues-and-discussions#238] [Как это сделать?] [answered]
Здравствуйте.
Подскажите, пожалуйста, со следующим вопросиком:
Есть телеграм бот, который управляется тотумом по webhooks. 
Отправка и прием сообщений работают ок.
Теперь нужно передать с сообщением еще Inline Keyboards (представляет собой глубокий JSON).
Проблема в следующем:
тотум отправляет JSON боту, бот выводит сообщение, но без переданных ему кнопок (структуру JSON проверил, все норм). В инете нашел мнение, что нужно указать глубину передаваемого боту JSON. Как указать глубину JSON в тотум? (думаю скорее всего это нужно делать на самом PHP)
Отправляемый JSON:
{
  "text": "текст сообщения",
  "chat_id": "531559635",
  "parse_mode": "HTML",
  "reply_markup":{"inline_keyboard": [[{"text": "Кнопка 1", "url": "https://t.me/t_video_net_bot"}]]}
}
Спасибо.

**Comment by erpic-ru** [comment, marked as answer]
Обсуждали уж эти злосчастные кнопки :)

https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/218

**Reply by tvideo** [reply]
биг спс
Работает
